lib/diaspora_federation/validators/rules/ids_count.rb

```diff
@@ -0,0 +1,27 @@
+module Validation
+  module Rule
+    # Rule for validating the number of Diaspora* ids in a string.
+    # The evaluated string is split at ";" and the result will be counted.
+    class IdsCount
```

Comment:
I think ID is too generic here, I would name it DiasporaIdCount (also singular, its TagCount not TagsCount).

Maybe you can also validate each of them against the regex in the DiasporaId-Rule?